src/app/tables/table.component.html

```diff
@@ -65,12 +65,12 @@ <h2 class="table-title" *ngIf="title">
                 <!-- column.value is a boolean determining if the column is selected to be shown by the user -->
                 <td *ngIf="column.value">
                   <!-- Check if the column should be a link -->
-                  <ng-container *ngIf="linkColumnMappings &&
-                                      uidColumn &&
-                                      linkColumnMappings[item[uidColumn]] !== undefined &&
-                                      linkColumnMappings[item[uidColumn]][column.key] !== undefined;
+                  <ng-container *ngIf="uidColumn &&
```

Comment:
This looks great. Can you move the `uidColumn` check to the `tableColumn` pipe instead of here. That's good for the tool tip as well.